app/grants/views.py

```diff
@@ -306,6 +307,7 @@ def grant_fund(request, grant_id,  grant_slug):
         subscription.token_address = request.POST.get('denomination', '')
         subscription.token_symbol = request.POST.get('token_symbol', '')
         subscription.gas_price = request.POST.get('gas_price', 0)
+        subscription.sub_new_approve_tx_id = request.POST.get('sub_new_approve_tx_id', '')
```

Comment:
We really need to migrate this to a form or something 🤔

Reply:
Agreed. Does this block this PR? I think this issue is meant to address this concern https://github.com/gitcoinco/web/issues/3101